modules/tinymce/src/core/test/ts/webdriver/keyboard/IframeTabfocusTest.ts

```diff
@@ -70,5 +69,15 @@ describe('webdriver.tinymce.core.keyboard.IframeTabfocusTest', () => {
       await RealMouse.pClickOn('iframe => body');
       assertIsHighlighted(editor);
     });
+
+    it('TINY-9277: Remove highlight when not in focus', async () => {
```

Comment:
This could also be done without using webdriver just focus something else on the page with element.focus() or do editor.focus(); you could keep this test as well but I prefer to avoid having webdriver tests if we don't need to.

Reply:
Agreed that we should avoid writing webdriver tests if possible. I have converted this into a browser test.